pom.xml

```diff
@@ -53,6 +53,7 @@
         <clowder-quarkus-config-source.version>1.0.0</clowder-quarkus-config-source.version>
 
         <cloudwatch.version>4.2.0</cloudwatch.version>
+        <sentry.version>1.1.0</sentry.version>
```

Comment:
Should be renamed something like `quarkus.sentry.version` because this is not the Sentry version here. (Which would be something like 5.x.x)

Reply:
Looks like we would upgrade from 5.5.0 to 5.6.3 (a good thing).
https://repo1.maven.org/maven2/io/quarkiverse/loggingsentry/quarkus-logging-sentry-parent/1.1.0/quarkus-logging-sentry-parent-1.1.0.pom
Sentry will still complain that we are not at 5.7.1 though :-)